Cargo.toml

```diff
@@ -15,4 +15,4 @@ bzip2 = "0.3"
 xz2 = "0.1"
 
 [dev-dependencies]
-pretty_assertions = "*"
+pretty_assertions = "*"
```

Comment:
What editor are you using? You might want to modify its settings to ensure it doesn't strip the final line terminator.